backend/app/main.py

```
@@ -41,15 +41,17 @@
     "http://localhost:3001",
     "http://localhost:3000",
     "https://handi-craft.xyz",
-    "https://www.handi-craft.xyz",
+    "https://www.handi-craft.xyz/"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)